app.go

```diff
@@ -42,14 +41,13 @@ func New(appID string, config *Config) (*App, error) {
 		client: client,
 	}
 
-	app.jwksCache = jwk.NewCache(context.Background())
-	if err := app.jwksCache.Register(fmt.Sprintf(jwksUrl, appID)); err != nil {
+	cache := jwk.NewCache(context.Background())
+	if err := cache.Register(fmt.Sprintf(jwksUrl, appID)); err != nil {
 		return nil, err
 	}
 
-	if err := app.refreshJWKSCache(); err != nil {
```

Comment:
the cache needs to be manually [refreshed](https://github.com/passageidentity/passage-go/pull/95/files#diff-f2af39c59ae7258ce6dfa1f7fea0c60d8213467642c8bb936ca8339559524057L65-L67) just once before initializing the cached set

[reference](https://github.com/lestrrat-go/jwx/blob/536b643c661a886300246d2f334d4d836e61f497/docs/04-jwk.md?plain=1#L699-L705) from the example

Reply:
Added back the refresh